backend/dataall/modules/loader.py

```diff
@@ -19,9 +19,9 @@ class ImportMode(Enum):
     of functionality to be loaded, there should be different loading modes
     """
 
-    API = "api"
-    CDK = "cdk"
-    TASKS = "tasks"
+    API = auto()
+    CDK = auto()
+    HANDLERS = auto()
 
```

Comment:
I like this, it defaults to the lowercase right?

Reply:
It's used to apply default values to enums. It can be often seen when the value is not important